### PR TITLE
Access Control and OIDC

### PR DESCRIPTION
**Pull request by hhund**
* New FHIR Server Access Control page
* New FHIR Server OIDC page
* Modifications / fixes to the install doc
* New "maintain" menu structure
* Replaced gmds tutorial announcement with talk

### CODE REVIEW

File: docs/src/.vuepress/theme.ts [hunk 1]
```diff
@@ -123,14 +123,44 @@ export default hopeTheme({
...
+			}]
+      }, {
+        text: "BPE Server",
+        icon: "module",
+        prefix: "bpe/",
+        link: "bpe/",
+        children: [{
+			icon: "config", 
+			text: "Configuration",
+			link:"configuration"
```
Comment (MadMax93) [inline docs/src/.vuepress/theme.ts:161]:
```suggestion
			link: "configuration"
```

File: docs/src/v1.1.0/maintain/fhir/access-control.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+title: Access Control
+icon: config
+---
+
+## Overview
+
+The DSF FHIR server implements a subset of the FHIR R4 [REST API](http://hl7.org/fhir/R4/http.html). When accessing the API with a web browser a limited graphical user interface is shown. Without any additional configuration the API and user interface is only accessible with the X.509 client certificate configured for the organization via the configuration parameter: [DEV_DSF_FHIR_SERVER_ORGANIZATION_THUMBPRINT](configuration#dev-dsf-fhir-server-organization-thumbprint)
+
+::: tip OpenID Connect
+To enable OpenID Connect Authentication of local user, see the DSF FHIR server OpenID Connect [configuration page](oidc).
```
Comment (MadMax93) [inline docs/src/v1.1.0/maintain/fhir/access-control.md:11]:
```suggestion
To enable OpenID Connect authentication of local user, see the DSF FHIR server OpenID Connect [configuration page](oidc).
```

File: docs/src/v1.1.0/maintain/fhir/access-control.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+            thumbprint: 00474993fa261b0225f93c5a66aa6fcc... [a-f0-9]{128}
+            dsf-role:
+              - READ
+              - SEARCH
+              - HISTORY
+```
+The list of user roles above contains a single entry `example_read_only_role`, matching the user via a client certificate SHA-512 thumprint and assigning three DSF roles.
+
+::: tip Certificate Thumbprints
+SHA-512 certificate thumbprints in the HEX form `[a-f0-9]{128}` can be calculated using:
```
Comment (MadMax93) [inline docs/src/v1.1.0/maintain/fhir/access-control.md:30]:
```suggestion
SHA-512 certificate thumbprints in HEX form `[a-f0-9]{128}` can be calculated using:
```

File: docs/src/v1.1.0/maintain/fhir/access-control.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+To apply roles, users can be matched via the `thumbprint`, `email`, `token-role` or `token-group` properties. A single value or a list of values can be specified.
+
+#### thumbprint
+
+The property `thumbprint` can used to specify one or multiple SHA-512 certificate thumbprints. Roles from this rule are applied to the authenticating user if the certificate matches one of the specified thumbprints.
+
+#### email
+
+Using the property `email` users can be matched against e-mail addresses specified in X.509 client certificates and in OpenID Connect access tokens. Values will be matched against e-mail addresses specified in the subject DN (via PKCS#9 extension 1.2.840.113549.1.9.1) and RFC-822 Name entries of the Subject Alternative Name field. If the user authenticates via OpenID Connect, the `email` [claim](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) from the access token will be matched against the property values.
```
Comment (MadMax93) [inline docs/src/v1.1.0/maintain/fhir/access-control.md:48]:
```suggestion
Using the property `email` users can be matched against e-mail addresses specified in X.509 client certificates and in OpenID Connect access tokens. Values will be matched against e-mail addresses specified in the subject DN (via PKCS#9 extension 1.2.840.113549.1.9.1) and RFC-822 name entries of the Subject Alternative Name field. If the user authenticates via OpenID Connect, the `email` [claim](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) from the access token will be matched against the property values.
```

File: docs/src/v1.1.0/maintain/fhir/access-control.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+#### token-role and token-group
+
+With the properties `token-role` and `token-group` role and group names can be specified to match agains role and group claims within OAuth 2.0 access tokens.
```
Comment (MadMax93) [inline docs/src/v1.1.0/maintain/fhir/access-control.md:52]:
```suggestion
With the properties `token-role` and `token-group` role and group names can be specified to match against role and group claims within OAuth 2.0 access tokens.
```

File: docs/src/v1.1.0/maintain/fhir/oidc.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+## Bearer Token Authentication
+
+To enable bearer token authentication, set the configuration parameter [DEV_DSF_SERVER_AUTH_OIDC_BEARER_TOKEN](configuration#dev-dsf-server-auth-oidc-bearer-token) to `true` and specify the following parameter:
+- [DEV_DSF_SERVER_AUTH_OIDC_PROVIDER_REALM_BASE_URL](configuration#dev-dsf-server-auth-oidc-provider-realm-base-url)
+
+
+## Additional ODIC Configuration Parameter
+
+A number of additional `DEV_DSF_SERVER_AUTH_OIDC ...` configuration parameter are specify on the DSF FHIR Server [configuration parameter page](configuration).
```
Comment (MadMax93) [inline docs/src/v1.1.0/maintain/fhir/oidc.md:36]:
```suggestion
A number of additional `DEV_DSF_SERVER_AUTH_OIDC ...` configuration parameter are specify on the DSF FHIR server [configuration parameter page](configuration).
```

**Review by MadMax93 — APPROVED** [review]

File: docs/src/v1.1.0/maintain/fhir/access-control.md (hunk 5 above)
Comment (hhund) [reply docs/src/v1.1.0/maintain/fhir/access-control.md]: fixed with 2a1a4d2

File: docs/src/v1.1.0/maintain/fhir/access-control.md (hunk 4 above)
Comment (hhund) [reply docs/src/v1.1.0/maintain/fhir/access-control.md:48]: No change, "Name" is part of the proper noun "RFC-822 Name"

File: docs/src/v1.1.0/maintain/fhir/access-control.md (hunk 3 above)
Comment (hhund) [reply docs/src/v1.1.0/maintain/fhir/access-control.md]: fixed with 2a1a4d2

File: docs/src/v1.1.0/maintain/fhir/access-control.md (hunk 2 above)
Comment (hhund) [reply docs/src/v1.1.0/maintain/fhir/access-control.md]: fixed with 2a1a4d2

File: docs/src/v1.1.0/maintain/README.md [hunk 7]
```diff
@@ -6,4 +6,11 @@ icon: tool
 - [Install DSF 1.0.0](install)
```
[the remark this answers is not in the document]
Comment (hhund) [reply docs/src/v1.1.0/maintain/README.md]: fixed with 2a1a4d2

File: docs/src/.vuepress/theme.ts (hunk 1 above)
Comment (hhund) [reply docs/src/.vuepress/theme.ts]: fixed with 2a1a4d2

File: docs/src/.vuepress/theme.ts [hunk 8]
```diff
@@ -123,14 +123,44 @@ export default hopeTheme({
...
+		}]
+      }, {
+        text: "FHIR Server",
+        icon: "module",
+        prefix: "fhir/",
+        link: "fhir/",
+        children: [{
+			icon: "config", 
+			text: "Configuration",
+			link:"configuration"
```
[the remark this answers is not in the document]
Comment (hhund) [reply docs/src/.vuepress/theme.ts]: fixed with 2a1a4d2

File: docs/src/.vuepress/theme.ts [hunk 9]
```diff
@@ -123,14 +123,44 @@ export default hopeTheme({
...
-      }
-     ],
+        text: "FHIR Reverse Proxy",
+        icon: "module",
+        prefix: "fhir-reverse-proxy/",
+        link: "fhir-reverse-proxy/",
+        children: [{
+			icon: "config", 
+			text: "Configuration",
+			link:"configuration"
```
[the remark this answers is not in the document]
Comment (hhund) [reply docs/src/.vuepress/theme.ts]: fixed with 2a1a4d2

**Review by schwzr — APPROVED** [review]

**Pull request merged**